src/state/slices/opportunitiesSlice/resolvers/uniV2/constants.ts

```diff
@@ -0,0 +1 @@
+export const FOX_TOKEN_CONTRACT_ADDRESS = '0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d'
```

Comment:
Exclusively used in LP domain 
<img width="850" alt="image" src="https://user-images.githubusercontent.com/17035424/219950318-86f6fd50-81a4-4504-aca3-486539908c3d.png">